pkg/server/app/fake_test.go

```diff
@@ -317,3 +317,34 @@ func TestFakeOperationsSetAutoscaletErrNotFound(t *testing.T) {
 		t.Errorf("expected ErrNotFound, got %v", err)
 	}
 }
+
+func TestFakeOperationsDelete(t *testing.T) {
+	fake := NewFakeOperations()
+	user := &database.User{Name: "gopher@luizalabs.com"}
+	app := &App{Name: "teresa"}
+	fake.(*FakeOperations).Storage[app.Name] = app
+
+	if err := fake.Delete(user, app.Name); err != nil {
+		t.Fatal("error on Delete: ", err)
```

Comment:
for the last test in a test function is ok to use `t.Error`, we use `t.Fatal` only if we want to stop the test.